openupgrade_scripts/scripts/website_sale/15.0.1.1/post-migration.py

```diff
@@ -1,6 +1,20 @@
 from openupgradelib import openupgrade
 
 
+def update_xpath_for_product_custom_text(env):
+    """Look for custom views website_sale.product_custom_text and update the content hook"""
+    for view in env["ir.ui.view"].search(
+        [
+            ("key", "=", "website_sale.product_custom_text"),
+            ("website_id", "!=", False),
+        ]
+    ):
+        view.arch_db = view.arch_db.replace(
```

Comment:
Is this working correctly in the website context?

Reply:
In this case we don't need the context as the script treats every custom view (check the domain)